### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Bedrock16** [kb24x7/musicbot#1]
Does railway allow us to play copyrighted music from YouTube?

**Comment by KunalBagaria** [comment]
Nope, I made the repository before it was restricted to host a musicbot. I'll remove the deploy to railway button soon.
